### ISSUE TITLE
LDK's fallback channel open procedure doesn't work with CLN

### ISSUE THREAD

**Issue by TonyGiorgio** [ElementsProject/lightning#6155]
### Issue and Steps to Reproduce
Whenever you try to open a SCID alias channel to CLN (without 0 conf) it fails because of this bug: https://github.com/ElementsProject/lightning/issues/5908

The way LDK handles that failure is by sending another `SendOpenChannel` message. However, it never proceedes and the peer connection gets disconnected after some time. What I expect is that the `FundingGenerationReady processing` step happens after it receives a response from CLN but that response never comes. I only seem to have this problem from LDK->CLN.

### `getinfo` output

I believe the CLN I'm trying this against is the december release.

**Comment by rustyrussell** [comment]
Yes, we hang up after that, since we don't like your message.  It's a bit gratuitous, but it's a bug now fixed anyway.

**Comment by rustyrussell** [comment]
AI cleanup of old issues.  If wrong, please open a new issue.


Triage decision summary:
LDK fallback channel open procedure fails with CLN due to a CLN bug (issue #5908) causing peer disconnection. Issue from April 2023 with no recent activity; likely fixed in subsequent CLN releases or workarounds implemented.
